src/tutorial/output.md

```diff
@@ -1,15 +1,136 @@
 # Output
 
+## Printing "Hello World"
+
+```rust
+println!("Hello World");
+```
+
+Well, that was easy.
+Great, onto the next topic.
+
+## Using println
+
+You can pretty much print all the things you like
+with the `println!` macro.
+This macro has some pretty amazing capabilities,
+but also a special syntax.
+It expects you to write,
+as a first parameter,
+a literal string that with placeholders,
+followed by the data you want to put into these placeholders
+as further arguments.
+
+For example:
+
+```rust
+let x = 42;
+println!("My lucky number is {}.", x);
+```
+
+will print
+
+> My lucky number is 42.
```

Comment:
supernit: you can use ```console

So it looks like this:

```console
My lucky number is 42.
```

Reply:
Same below

Reply:
I <3 super nits, thanks!